add highlights for 2023-08-31 (#1899)
giantswarm/docs: src/content/changes/highlights/2023-08-31.md

---
date: 2023-08-31T14:00:00
title: Highlights for the week ending August 31 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## Nginx Ingress Controller 

## Apps

> - [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
> - [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v1.1.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to Linkerd v2.13.6.
> - [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v1.1.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to Linkerd v2.13.6.
> - [linkerd-viz](https://github.com/giantswarm/linkerd-viz-app) version [v1.3.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#130---2023-08-29), upgrades to Linkerd v2.13.6 and brings better defaults for production environments and pod distribution budgets.
> 
> For more Information about the Linkerd v2.13.6 changes, please take a look at the [official release notes](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6)

Reviewer: ```suggestion
- [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
- [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v1.1.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to [Linkerd v2.13.6](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6).
- [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v1.1.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to [Linkerd v2.13.6](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6).
- [linkerd-viz](https://github.com/giantswarm/linkerd-viz-app) version [v1.3.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#130---2023-08-29), upgrades to [Linkerd v2.13.6](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6) and brings better defaults for production environments and pod distribution budgets.

For more information about the Linkerd v2.13.6 changes, please take a look at the [official release notes](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6)
```

> 

Reviewer 2: ```suggestion
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.15.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0151---2023-08-16)(and v0.15.0) updates to the highly anticipated Kyverno version 1.10.2. This release brings major architectural changes to the Kyverno controllers as well as breaking changes to the upstream Helm chart. During the upgrade to 0.15.0 or 0.15.1, existing Kyverno deployments will be briefly scaled to 0 and replaced with the new version. **Important**: The Helm schema has changed to reflect the new deployment structure, so if you are overriding Helm values, review the [release notes](https://github.com/kyverno/kyverno/releases/tag/v1.10.0) and [upgrade guide](https://github.com/kyverno/kyverno/blob/release-1.10/charts/kyverno/README.md#migrating-from-v2-to-v3) to ensure any relevant configuration will still apply to the new controllers. These changes include significant stability and performance improvements and VPA support for more Kyverno components.
```

---
date: 2023-08-31T14:00:00
title: Highlights for the week ending August 31 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## aws-load-balancer
- [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) version [v1.3.4](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#134---2023-08-28), migrates from monitoring labels to ServiceMonitor and introduces a new psp for CAPA support.

> ## external-dns
> - [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.

Reviewer: ```suggestion

## external-dns
- [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
```

---
date: 2023-08-31T14:00:00
title: Highlights for the week ending August 31 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

> ## aws-load-balancer
> - [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) version [v1.3.4](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#134---2023-08-28), migrates from monitoring labels to ServiceMonitor and introduces a new psp for CAPA support.

Reviewer: ```suggestion
## aws-load-balancer
- [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) version [v1.3.4](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#134---2023-08-28), migrates from monitoring labels to ServiceMonitor and introduces a new Pod Security Policy for Cluster API support (versions behind Kubernetes 1.25).
```

## linkerd
- [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v1.1.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to Linkerd v2.13.6.
- [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v1.1.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to Linkerd v2.13.6.
- [linkerd-viz](https://github.com/giantswarm/linkerd-viz-app) version [v1.3.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#130---2023-08-29), upgrades to Linkerd v2.13.6 and brings better defaults for production environments and pod distribution budgets.

For more Information about the Linkerd v2.13.6 changes, please take a look at the [official release notes](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6)

## ingress-nginx

> - We spent the last months reworking our [ingress-nginx-app](https://github.com/giantswarm/ingress-nginx-app) by aligning it to the [upstream ingress-nginx](https://github.com/kubernetes/ingress-nginx) project. This is necessary to be future-proof, feature compliant, and offer the best ingress experience possible.. Therefore we want to announce the [first public stable release](https://github.com/giantswarm/ingress-nginx-app/blob/main/CHANGELOG.md#300---2023-08-28) of our new ingress-nginx chart.
> If you are currently using `v2.x.x`, this release includes breaking changes. We set up a [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0/migration.md) to make the upgrade as smooth as possible. Notable changes requiring your attention and/or manual intervention, like renaming, deprecation or removal of values, have been highlighted below.
> Even though we highly recommend to upgrade to this and future releases, `v2.x.x` will continue to receive bugfixes as long as possible.
> 
> All feedback regarding this release, its changes, or our [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0/migration.md) is very welcome!

Reviewer: ```suggestion
- We spent the last months reworking our [ingress-nginx-app](https://github.com/giantswarm/ingress-nginx-app) by aligning it to the [upstream ingress-nginx](https://github.com/kubernetes/ingress-nginx) project. This is necessary to be future-proof, feature compliant, and offer the best ingress experience possible. Therefore we want to announce the [first public stable release](https://github.com/giantswarm/ingress-nginx-app/blob/main/CHANGELOG.md#300---2023-08-28) of our new ingress-nginx chart.
This release includes breaking changes if you are currently using `v2.x.x`. We set up a [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0/migration.md) to make the upgrade as smooth as possible. Notable changes requiring your attention and/or manual intervention, like renaming, deprecating or removing values, have been highlighted below.
Even though we highly recommend upgrading to this and future releases, `v2.x.x` will continue receiving bug fixes as long as possible.

All feedback regarding this release, its changes, or our [migration guide](https://github.com/giantswarm/ingress-nginx-app/blob/v3.0.0/migration.md) is very welcome!
```
